Source/WebCore/page/PerformanceNavigationTiming.cpp

```diff
@@ -140,7 +140,7 @@ double PerformanceNavigationTiming::duration() const
     return loadEventEnd() - startTime();
 }
 
-void PerformanceNavigationTiming::navigationFinished(const NetworkLoadMetrics& metrics)
+void PerformanceNavigationTiming::updateNetworkLoadMetrics(const NetworkLoadMetrics& metrics)
```

Comment:
It's probably better to call this `documentLoadFinished` to match the caller and make it clear what the update implies - that something has finished.